source/_integrations/group.markdown

```diff
@@ -18,24 +25,184 @@ ha_platforms:
   - notify
 ---
 
-Groups allow the user to combine multiple entities into one.
+The group integration lets you combine multiple entities into a single entity. Entities that are members of a group can be controlled and monitored as a whole.
 
-Check **Developer Tools** -> **States** and browse the **Current entities:** listing for all available entities.
+This can be useful for cases where you want to control, for example, the
+multiple bulbs in a light fixture as a single light in Home Assistant.
+
+Home Assistant can group multiple binary sensors, covers, fans, light, media players as a single entity, with the option of hiding the individual member entities.
+
+{% include integrations/config_flow.md %}
+
+## Group behavior
+
+By default, when any group member entity is `on`, the group will also be `on`. A complete overview of how groups behave:
+
+- The group state is `unavailable` if all group members are `unavailable`.
+- Otherwise, group state is `unknown` if all group members are `unknown`.
+- Otherwise, group state is `on` if at least one group member is `on`.
```

Comment:
Can we move this to be the first entry in the list

Reply:
Probably for the section below too. Seems like it should first say how `on` and `off` works, then talk about the more exceptional cases of `unknown` and `unavailable`